tests/tracer/test_span.py

```diff
@@ -479,6 +480,45 @@ def test_span_unicode_set_tag():
     span._set_str_tag(u"😐", u"😌")
 
 
+@pytest.mark.skipif(sys.version_info.major != 2, reason="This test only applies Python 2")
+@mock.patch("ddtrace.span.log")
+def test_span_binary_unicode_set_tag(span_log):
+    span = Span(None, None)
+    span.set_tag("key", "🤔")
+    span._set_str_tag("key_str", "🤔")
+    # only span.set_tag() will fail
+    span_log.warning.assert_called_once_with("error setting tag %s, ignoring it", "key", exc_info=True)
+    assert "key" not in span.meta
+    assert span.meta["key_str"] == u"🤔"
+
+
+@pytest.mark.skipif(sys.version_info.major >= 2, reason="This test does not apply to Python 2")
```

Comment:
This test case will never run?

Reply:
either the reason statement is wrong or the condition.

Reply:
I suspect it was meant to be `<= 2`